packages/ssr/src/createBrowserClient.ts

```diff
@@ -60,13 +66,15 @@ export function createBrowserClient<
 					if (isBrowser()) {
 						document.cookie = serialize(key, value, {
 							httpOnly: false,
```

Comment:
We could also move `httpOnly: false` to our DEFAULT_COOKIE_OPTIONS`

Reply:
Good idea.